dist/docker-images/ziti-cli/Dockerfile

```diff
@@ -21,6 +21,8 @@ ARG TARGETARCH
 ARG ZUID=2171
 ARG ZGID=2171
 
+ARG HOME=/home/ziggy
```

Comment:
small changes like this might cause issues downstream for anything using this container. i'd rather not make this particular change (which means the ones below need to change back too)

Reply:
It should have been this way since we enabled CLI auto-complete, which involved defining a HOME dir. In other words, this expresses one of the assumptions that's built in to the container image. This is a default value, not an override, so anyone who happens to be using this container with some other value for HOME will not be affected. This can only cause things to start working correctly, not break.